dev/com.ibm.ws.grpc_fat/build.gradle

```diff
@@ -30,7 +30,7 @@ buildscript {
     }
     dependencies {
         if (project.hasProperty('compileProtobufs')) {
-            classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.12'
+            classpath 'com.google.protobuf:protobuf-gradle-plugin:0.9.4'
```

Comment:
IMO updating the copyright date range to include 2023 is appropriate here.

Reply:
Done

Reply:
Still has 2022

Reply:
2023 now